config/sevntu-suppressions.xml

```diff
@@ -46,4 +46,11 @@
   <!-- Require for testing of util package -->
   <suppress checks="ForbidCertainImports" files="AnnotationUtilTest\.java"/>
   <suppress checks="ForbidCertainImports" files="CheckUtilTest\.java"/>
+
+  <!-- Require for code optimization -->
+  <suppress checks="MultipleStringLiteralsExtended" files="Checker\.java"/>
+  <suppress checks="MultipleStringLiteralsExtended" files="AbstractFileSetCheck\.java"/>
```

Comment:
Not ok, please move all constants that are duplicated in single file to constant.

Reply:
suppression removed